src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -116,7 +138,7 @@
           DependsOnTargets="DiscoverTestInputs;CheckTestCategories"
           AfterTargets="CheckTestCategories"
           Inputs="@(RunTestsForProjectInputs)"
-          Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestTargetFramework.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
+          Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestNugetTargetMoniker.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
```

Comment:
I think we need to refactor TestNuGetTargetMoniker.  It's used incorrectly today, some places it's treated as a batch, others (eg: package resolution from lock file) its treated as a fallback list.  I don't think we should be doing that refactoring as part of this change.